client/ayon_core/pipeline/project_folders.py

```diff
@@ -1,6 +1,7 @@
 import os
```

Comment:
With this you can replace `List` > `list` and `Dict` > `dict`.
```suggestion
from __future__ import annotations
import os
```

Reply:
Like this? https://github.com/ynput/ayon-core/pull/1169/commits/8008ab02b543b2d42a0dfbb91bd6afca315d28cf

Reply:
Yep